wrappers/null/popcorn.HTMLNullVideoElement.js

```diff
@@ -134,8 +134,12 @@
         callback();
       }
 
+      function isMobile() {
+        return navigator.userAgent.match(/(iPad|iPhone|iPod|Android)/g);
```

Comment:
@mstmustisnt, didn't receive bugreports from users about WP. Could you please give a feedback about it if you have some time?